src/index.js

```diff
@@ -1 +1,22 @@
-console.log('Hello');
\ No newline at end of file
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    prompt: 'Type a string to reverse: '
+});
+
+const printString = function (str) {
```

Comment:
Consider using arrow functions everywhere if you're not using "this" in your logic.

Reply:
done.